src/components/form/linkButton.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components'
+import { LocalizedLink } from 'components/localization'
+
+const LinkButton = styled(LocalizedLink)`
```

Comment:
LinkButtons can also be `primary, secondary, tertiary`? i'd suggest moving the component inside button.js and share the css between button and LinkButton